dags/utility/utility_index_from_s3.py

```diff
@@ -0,0 +1,188 @@
+"""
+## Utility Tool
+### update from S3 to Datacube
+This is utility is to provide administrators the easy accessiblity to run s3-to-dc
+
+#### Utility utilization
+The DAG must be parameterized with run time configurations `s3_glob` and `products`
+In addiction the DAG can receive the parameters `no_sign_request` and , `stac`
+
+To run with all, set `dag_run.conf.products` to `--all`
+otherwise provide list of products, to be refreshed, separated by space,
+i.e. `s2_l2a ls5_sr ls5_st ls7_sr ls7_st`
+
+dag_run.conf format:
+
+#### example conf in json format
+    {
+        "s3_glob":"s3://deafrica-sentinel-2-dev/sentinel-s2-l2a-cogs/**/*.json",
+        "product_definition_uri":"https://raw.githubusercontent.com/digitalearthafrica/config/master/products/ls5_c2l2.odc-product.yaml",
+        "products":"s2_l2a ls5_sr ls5_st ls7_sr ls7_st",
+        "no_sign_request":true,
```

Comment:
why `no_sign_request` conf? what happen if I pass `"no_sign_request": false` I mean

Reply:
do you then expect user creds?

Reply:
actually, you are right, if the flag isn't there or is false, it would require credentials that I haven't added a parameter for that. So I'll remove that and leave that as default